src/Tests/Tests/Analysis/Analyzers/AnalyzerTests.cs

```diff
@@ -69,14 +69,14 @@ public class SimpleTests : AnalyzerAssertionBase<SimpleTests>
 			public override string Name => "mySimple";
 		}
 
-		public class LanguageTests : AnalyzerAssertionBase<SimpleTests>
+		public class LanguageTests : AnalyzerAssertionBase<LanguageTests>
 		{
 			public override FuncTokenizer Fluent => (n, an) => an
 				.Language("myLanguage", a => a.Language(Language.Dutch));
 
 			public override IAnalyzer Initializer => new LanguageAnalyzer { Language = Language.Dutch };
 
-			public override object Json => new { type = "dutch" };
+			public override object Json => new { type = "Dutch" };
```

Comment:
There is one test failure because of this

Reply:
d'oh, thanks